csi.proto

```diff
@@ -1412,6 +1437,15 @@ message NodeServiceCapability {
       // Note that, for alpha, `VolumeCondition` is intended to be
       // informative for humans only, not for automation.
       VOLUME_CONDITION = 4 [(alpha_enum_value) = true];
+      // Indicates that Node service supports mounting volumes
+      // with provided volume group identifier during node stage
+      // or node publish RPC calls.
+      // It is expected that SP SHOULD use provided volume_mount_group
```

Comment:
We have to say here that if the capability is asserted, and the CO passes a non-nil group, then it MUST be respected. I don't think SHOULD is the right language here.

Reply:
reworded.